apps/meteor/client/providers/TranslationProvider.tsx

```diff
@@ -26,7 +25,7 @@ import {
 import { AppClientOrchestratorInstance } from '../../ee/client/apps/orchestrator';
 import { isRTLScriptLanguage } from '../lib/utils/isRTLScriptLanguage';
 
-i18n.use(I18NextHttpBackend).use(initReactI18next).use(sprintf);
+i18n.use(I18NextHttpBackend).use(initReactI18next);
```

Comment:
is it expected to remove the `sprintf` middleware?

Reply:
`apps/meteor/app/utils/lib/i18n.ts` already use `sprintf`